python/requirements.txt

```diff
@@ -1,5 +1,5 @@
 pip==9.0.3
-setuptools==54.0.0
+setuptools==44.0.0
```

Comment:
not sure why this wont let me set newer setuptools - versions available only seem to go to 44.1

Reply:
Version 44 was the last to support both Python 2 & 3. So until rules_python fully drops Python 2 support, it may be stuck with this version.